platform/jewel/samples/showcase/src/main/kotlin/org/jetbrains/jewel/samples/showcase/components/ComboBoxes.kt

```diff
@@ -0,0 +1,268 @@
+// Copyright 2000-2025 JetBrains s.r.o. and contributors. Use of this source code is governed by the
+// Apache 2.0 license.
+package org.jetbrains.jewel.samples.showcase.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.input.rememberTextFieldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import org.jetbrains.jewel.ui.component.EditableComboBox
+import org.jetbrains.jewel.ui.component.EditableListComboBox
+import org.jetbrains.jewel.ui.component.GroupHeader
+import org.jetbrains.jewel.ui.component.ListComboBox
+import org.jetbrains.jewel.ui.component.PopupContainer
+import org.jetbrains.jewel.ui.component.PopupManager
+import org.jetbrains.jewel.ui.component.SimpleListItem
+import org.jetbrains.jewel.ui.component.Text
+import org.jetbrains.jewel.ui.icon.IconKey
+import org.jetbrains.jewel.ui.icons.AllIconsKeys
+
+@Composable
+public fun ComboBoxes() {
+    GroupHeader("List combobox")
+    ListComboBoxes()
+
+    Spacer(modifier = Modifier.height(16.dp))
+
+    GroupHeader("Simple combobox")
+    SimpleComboBoxes()
+}
+
+@Composable
+private fun ListComboBoxes() {
+    val comboBoxItems = remember {
+        listOf(
+            "Cat",
+            "Elephant",
+            "Sun",
+            "Book",
+            "Laughter",
+            "Whisper",
+            "Ocean",
+            "Serendipity lorem ipsum",
+            "Umbrella",
+            "Joy",
+        )
+    }
+
+    val languageOptions = remember {
+        listOf(
+            ProgrammingLanguage("Java", AllIconsKeys.FileTypes.Java),
```

Comment:
What about Kotlin? 🤣

Reply:
Also, both this and `comboBoxItems` can be made top-level private vals, so there is no need for remember

Reply:
I couldn't find the icon 🤣
![image](https://github.com/user-attachments/assets/bd24ac52-c943-4c9c-9b5e-c8dfe0feccd6)